lib/components/settings/layout/font/font_settings_screen.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:harpy/components/components.dart';
+import 'package:harpy/components/settings/layout/font/widgets/preview_tweet_card.dart';
+import 'package:harpy/core/core.dart';
+
+class FontSettingsScreen extends StatefulWidget {
+  const FontSettingsScreen();
+
+  static const String route = 'font_settings';
+
+  @override
+  _FontSettingsScreenState createState() => _FontSettingsScreenState();
+}
+
+class _FontSettingsScreenState extends State<FontSettingsScreen> {
+  final LayoutPreferences layoutPreferences = app<LayoutPreferences>();
+
+  List<Widget> _buildSettings(ThemeBloc themeBloc) {
+    return <Widget>[
+      PreviewTweetCard(),
+      ListTile(
+        leading: const Icon(CupertinoIcons.textformat_size),
+        title: const Text('font size delta'),
+        subtitle: Slider(
+          value: layoutPreferences.fontSizeDelta,
+          onChanged: (newValue) {
+            HapticFeedback.lightImpact();
+            setState(() {
+              layoutPreferences.fontSizeDelta = newValue;
+              themeBloc.add(const RefreshTheme());
+            });
+          },
```

Comment:
For some reason this is triggered multiple times when changing the value.
Easy fix would be to check whether the value changed from the active value

```suggestion
          onChanged: (newValue) {
            if (layoutPreferences.fontSizeDelta != newValue) {
              HapticFeedback.lightImpact();
              setState(() {
                layoutPreferences.fontSizeDelta = newValue;
                themeBloc.add(const RefreshTheme());
              });
            }
          },
```